app/src/main/java/com/fs_sournary/weather/screen/main/MainActivity.java

```diff
@@ -1,7 +1,13 @@
 package com.fs_sournary.weather.screen.main;
 
+import android.Manifest;
+import android.content.pm.PackageManager;
 import android.databinding.DataBindingUtil;
 import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v4.app.ActivityCompat;
+import android.support.v4.content.ContextCompat;
+import android.widget.Toast;
```

Comment:
ERROR: Unused import: android.widget.Toast. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports